### PR TITLE
Default to ssh-ng and and builders-use-substituters

### PR DESCRIPTION
**Pull request by phaer**

### CODE REVIEW

**Comment by phaer** [comment]
Ah right, I forgot that ssh-ng still has a few unsolved complexities in relation to other flags, as described for example in https://github.com/NixOS/nix/pull/5010

File: src/nixos-anywhere.sh [hunk 1]
```diff
@@ -46,6 +46,8 @@ Options:
   URL of the source Nix store to copy the nixos and disko closure from
 * --build-on-remote
   build the closure on the remote machine instead of locally and copy-closuring it
+* --legacy-ssh
```
Comment (Mic92) [inline src/nixos-anywhere.sh:49]: Maybe:

```suggestion
* --builder-protocol
```

Than we can also support other protocols in future?

**Comment by Mic92** [comment]
@

> @phaer any chance you'd want to make it an opt-in feature for now?

**Comment by Mic92** [comment]
Still working on this?

**Comment by phaer** [comment]
Ah no, sorry for leaving this as a draft for so long & thanks for going through stalled PRs :) . I don't plan to set up too many new hosts and don't currently have time for it.

**Pull request closed without merging**